src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerCosmosSqlDependencyExtensions.cs

```diff
@@ -78,13 +78,81 @@ public static partial class ILoggerExtensions
         /// Logs a Cosmos SQL dependency.
         /// </summary>
         /// <param name="logger">The logger to track the telemetry.</param>
-        /// <param name="accountName">Name of the storage resource</param>
-        /// <param name="database">Name of the database</param>
-        /// <param name="container">Name of the container</param>
-        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
-        /// <param name="startTime">Point in time when the interaction with the dependency was started</param>
-        /// <param name="duration">Duration of the operation</param>
-        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        /// <param name="accountName">The name of the storage resource.</param>
+        /// <param name="database">The name of the database.</param>
+        /// <param name="container">The name of the container.</param>
+        /// <param name="isSuccessful">The indication whether or not the operation was successful</param>
+        /// <param name="measurement">The measuring the latency of the dependency.</param>
```

Comment:
```suggestion
        /// <param name="measurement">The measurement of the duration of the dependency.</param>
```

Reply:
Done in https://github.com/arcus-azure/arcus.observability/pull/380/commits/6b0d20fcd4b0765317effe2c60a582a2b981e1f4